src/app/projects/states/dashboard/directives/task-dashboard/task-dashboard.coffee

```diff
@@ -35,11 +35,12 @@ angular.module('doubtfire.projects.states.dashboard.directives.task-dashboard',
       $scope.urls = {
         taskSheetPdfUrl: Task.getTaskPDFUrl($scope.task.unit(), $scope.task.definition)
```

Comment:
Given we have the task list to view task sheets, has this been updated to include the attachment options? This would mean changes to the backend as well.